summer-qa-task-2024/src/e2e/tests/addItem.test.ts

```diff
@@ -0,0 +1,116 @@
+import { test, expect } from "@playwright/test";
+import { TaskManager } from "../components/ui/TaskManager";
+import { TaskLists } from "../components/ui/TaskLists";
+import { TaskItem } from "../components/ui/TaskItem";
+import { urls } from "../components/urls";
+import path from "path";
+import fs from "fs";
+
+test.describe("Task Manager - Add Task", () => {
+  let taskManager: TaskManager;
+  let taskLists: TaskLists;
+
+  const screenshotsDir = path.join(process.cwd(), "failed-screenshots");
+
+  if (!fs.existsSync(screenshotsDir)) {
```

Comment:
curios de ce e nevoie sa faci asta ? playwright nu face by default fisierul in caz ca el nu este?